.gitignore

```diff
@@ -8,3 +8,4 @@ nginx/sourcegraph.key
 .vagrant
 *.log
 
+ **/*.dhall 
```

Comment:
@ElizabethStirling I'm not sure this PR (or the one in deploy-sourcegraph) makes sense. Dhall is just a language, akin to YAML or JSON - so ignoring _all_ Dhall files seems quite wrong to me.

I'm fine with moving forward with this if you think it will help and then later on restricting this further to a specific Dhall file name or something, though maybe we could do that now instead?

Reply:
Will update it to only ignore the single, specific file that we had issues with.